.github/workflows/move-ready-for-review-prs.yml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+name: Move PR To Initial Review
+on:
+  issue_comment:
+    types: [created]
+    branch:
+      - main
+
+jobs:
+  move-pr-to-initial-review:
+    if: github.event.comment.body == '/pr mark ready' && github.event.issue.pull_request
+    runs-on: ubuntu-latest
+    steps:
+      - name: Move To Initial Review
+        uses: actions/github-script@v6
+        with:
+          script: |
+            // Find "Code Reviews" project manually by name matching
+            // This avoids hardcoding the project ID
+            const projects = await github.rest.projects.listForRepo({
+              owner: context.repo.owner,
+              repo: context.repo.repo,
+            })
+            const code_reviews = projects.data.find(project => {
+              return project.name === 'Code Reviews';
+            })
+            if (!code_reviews) {
+              return

```

Comment:
It'd be nice to log if this happens (same below)

Reply:
Done.